docs/docsite/rst/command_guide/cheatsheet.rst

```diff
@@ -121,4 +121,31 @@ ansible-doc
 .. code-block:: bash
 
   ansible-doc -t module -l
-  #...
\ No newline at end of file
+  #...
+
+ansible-config
+==============
+The ansible-config command in Ansible is used to view or modify Ansible configuration settings. 
+
+.. code-block:: bash
+
```

Comment:
The formatting is still off.  See https://ansible--1482.org.readthedocs.build/projects/ansible-core/en/1482/command_guide/cheatsheet.html#ansible-config

I think the fix is to indent everything below this line by 2 spaces. Basically the code-block directive expects everything that is to be in that code-block to be indented.